docs/cndb_omf.md

```diff
@@ -0,0 +1,294 @@
+DRAFT!!! DRAFT!!! DRAFT!!! DRAFT!!! DRAFT!!! DRAFT!!!
+
+# OMF descriptions
+
+Describes the proposed CNDB omf for HSE-3.0.
+
+## Record types
+
+Each record type contains a header at the beginning. This header contains the type of the record and its
+size. This header is not listed in the following breakdown of the various record types.
+
+### Version
+- Magic: A magic number to verify that the MDC is a CNDB MDC.
+- Version: Version of the CNDB omf.
+- Captgt: (Capacity Target) Size of the CNDB.
+
+### Info
+There are two types of info records. `CNDB_TYPE_INFO` and `CNDB_TYPE_INFOD`.
+- Create-time parameters for a KVS
+  - Prefix length: Length of prefixes when using prefix delete.
+- Flags: Flags for the KVS
+  - `CN_CFLAG_CAPPED`: Whether or not the KVS is a capped KVS.
+- Meta size: The Info record is followed by opaque metadata. This field records the size of this metadata.
+  See the "Meta[]" field below.
+- Cnid: (CN id) A unique identifier for the KVS.
+- CN name: Name of the KVS
+- Meta[]: Opaque metadata ecoded/decoded by the caller.
+
+### CNDB meta
+- Seqno max: Max seqno of the persisted records in the KVDB.
```

Comment:
Seqno of records?  Or seqno of key-value pairs in the KVDB?

Also, what is the seqno used for?

Reply:
Seqnos only apply to kv-pairs. It keeps track of the largest seqno that has been persisted when this record is written. It serves as the starting point for cndb->cndb_seqno at replay.